### PR TITLE
Set service state to enabled for contiv services

### PR DESCRIPTION
**Pull request by neelimamukiri**
Fixes https://github.com/contiv/ansible/issues/304
Also setting the Install section to allow enabling the services.

### CODE REVIEW

**Comment by unclejack** [comment]
LGTM

@DivyaVavili @gaurav-dalvi: Could you review this when you have a chance, please?

File: roles/contiv_network/tasks/main.yml [hunk 1]
```diff
@@ -82,8 +85,12 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
+- name: reload system config for netmaster
+  shell: systemctl daemon-reload
+  when: run_as == "master"
```
Comment (dvavili) [inline roles/contiv_network/tasks/main.yml]: Why is this change required?

Comment (neelimamukiri) [reply roles/contiv_network/tasks/main.yml]: Why do the reload before setting the state to started/enabled? If the service is not loaded, apparently setting it to enabled fails.

Comment (dvavili) [reply roles/contiv_network/tasks/main.yml]: OK. Could you use the daemon-reload option in service module rather than have a separate task for this?

(Ref: https://docs.ansible.com/ansible/systemd_module.html#options)

Comment (dvavili) [reply roles/contiv_network/tasks/main.yml]: Also, wouldn't we require the deamon-reload to happen even for etcd and aci-gw services?

**Review by dvavili — CHANGES_REQUESTED** [review]

File: roles/contiv_network/tasks/main.yml (hunk 1 above)
Comment (neelimamukiri) [reply roles/contiv_network/tasks/main.yml]: Sure, I can make that change as well. I wasn't really adding or changing the daemon-reload option, so I didn't check etcd and the other services. I will check and add that change along with this.

Comment (neelimamukiri) [reply roles/contiv_network/tasks/main.yml]: Given that etcd doesn't have the reload and still works fine, I wonder if it is required at all. I don't see any harm in adding it though, so will go ahead with it.

**Review by dvavili — APPROVED** [review]

**Comment by dvavili** [comment]
Thanks for incorporating the review comments. Changes LGTM.

**Comment by gaurav-dalvi** [comment]
LGTM. Very good and very quick @neelimamukiri . Keep it up.

**Comment by gaurav-dalvi** [comment]
Thanks @neelimamukiri .
@unclejack suggested these cleanup steps . Is it possible to incorporate those into our ansible scripts ?

```
sudo rm -rf ansible .gen .ansible genInventoryFile.py
sudo docker kill -s 9 $(sudo docker ps -q)
sudo docker rm -f $(sudo docker ps -a -q)
sudo systemctl stop docker
sudo systemctl stop etcd
for i in $(mount | grep docker | awk '{ print $3 }'); do sudo umount $i || true; done
sudo umount /var/lib/docker/devicemapper || true
sudo yum remove -y docker-engine.x86_64 docker-engine-selinux.noarch
sudo rm -rf /usr/bin/etcd*
sudo rm -rf /var/lib/etcd*
sudo rm -rf /var/lib/docker
```

**Comment by unclejack** [comment]
@gaurav-dalvi This PR only makes changes needed to allow the cluster to start after reboot. Cleanup will be a separate thing.

**Comment by neelimamukiri** [comment]
@gaurav-dalvi, lets add a different issue to track the clean up - current task cleanup is only stopping services and not truly uninstalling anything.
